frontend/app/controllers/SiteMap.scala

```diff
@@ -10,41 +9,41 @@ import scala.xml.Elem
 
 object SiteMap extends Controller with LazyLogging {
 
-  val countrySpecificGroups: Map[CountryGroup, Country] = (for {
-    countryGroup: CountryGroup <- ActiveCountryGroups.all
-    defaultCountry <- countryGroup.defaultCountry
-  } yield countryGroup -> defaultCountry).toMap
-
   def sitemap() = CachedAction { implicit request =>
     val foo = <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
                       xmlns:xhtml="http://www.w3.org/1999/xhtml">
       {supporterPages}
       <url>
         <loc>{routes.FrontPage.index.absoluteURL(secure = true)}</loc>
         <priority>0.8</priority>
+        {alternatePage(routes.FrontPage.index.absoluteURL(secure = true), "x-default")}
       </url>
     </urlset>
     Ok(foo)
   }
 
-  def supporterPages()(implicit req: RequestHeader): Iterable[Elem] = for {
-    countryGroup <- countrySpecificGroups.keys
+  private def supporterPages()(implicit req: RequestHeader): Iterable[Elem] = for {
+    countryGroup <- ActiveCountryGroups.all
   } yield {
     <url>
       <loc>
         {routes.Info.supporterFor(countryGroup).absoluteURL(secure = true)}
       </loc>
       <priority>1.0</priority>
-      {alternatePages()}
+      {alternatePages(ActiveCountryGroups.all)}
+      {alternatePage(routes.Info.supporterFor(countryGroup).absoluteURL(secure = true), "x-default")}
```

Comment:
No you're right, there is no true un-localised page, so I'll remove the x-default alternate.